File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def GCD(a, b):
    while b:
        a, b = b, a%b
    return a

def LCM(a, b):
    return (a*b) // GCD(a, b)

def solution(n, m):
    return [GCD(n, m), LCM(n, m)]

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
  const GCD = (a, b) => {
    while(b){
      [a, b] = [b, a%b]
    }
    return a
  }

  const LCM = (a, b) => {
    return (a*b) / GCD(a, b)
  }

  return [GCD(n, m), LCM(n, m)]
}